pint/utils.py

```diff
@@ -439,6 +440,74 @@ def is_number(s):
         pass
     return False
 
+def make_toas(startMJD, endMJD, ntoas, model, freq=1400, obs='GBT'):
```

Comment:
Perhaps it would be better to default to infinite frequency and the barycenter? This will give headaches if, for example, the user does not have up-to-date GBT clock corrections but isn't using GBT data anyway.

Reply:
TODO added